src/lib.rs

```diff
@@ -1,8 +1,60 @@
 //! A Rust implementation of the [GeoArrow](https://github.com/geoarrow/geoarrow) specification,
-//! plus algorithms implemented on and returning these GeoArrow arrays.
+//! including algorithms implemented on and returning these GeoArrow arrays.
+//!
+//! # Reading and writing
+//!
+//! The [io] module has functions for reading and writing GeoArrow data from a variety of formats.
+//! To use most format readers and writers, you must enable their corresponding feature.
+//! For example, to convert between [geosjon](https://geojson.org/) and GeoArrow, enable the `geozero` feature in your `Cargo.toml`:
+//!
+//! ```toml
+//! #[dependencies]
```

Comment:
Is there a reason for this `#`?
```suggestion
//! [dependencies]
```

Reply:
Nope, pure habit from rust's `#[derive]`, good catch